app/src/main/java/com/x8bit/bitwarden/ui/vault/feature/vault/VaultScreen.kt

```diff
@@ -142,6 +143,10 @@ private fun VaultScreenPushNotifications(
     permissionsManager: PermissionsManager,
 ) {
     if (Build.VERSION.SDK_INT < Build.VERSION_CODES.TIRAMISU) return
+    if (isFdroid) {
```

Comment:
Would it make sense to pass this in so this can be tested?

Reply:
💭 apples and oranges but along that line: you could combine the checks for the build version || FDroid?